pkg/detectors/nubela/nubela_test.go

```diff
@@ -0,0 +1,117 @@
+
```

Comment:
Please add the tags back. You can run the test by `go test ./pkg/detectors/nubela -tags=detectors`
```
//go:build detectors
// +build detectors
```